n_i2c.c

```diff
@@ -328,6 +355,12 @@ const char *i2cChunkedReceive(uint8_t *buffer, uint32_t *size, bool delay, size_
         // Add requested bytes to received total
         received += requested;
 
+        // Once we've received any character, we will no longer wait patiently
+        if (requested != 0) {
+            timeoutMs = (NOTECARD_INTRA_TRANSACTION_TIMEOUT_SEC * 1000);

```

Comment:
When `i2cChunkedReceive` is called in `i2cNoteTransaction`, it is called with `timeoutMs` already set to 1 second. The other place it gets called is `NoteBinaryStoreReceive`, from what I can tell, and it's also called with 1 second there. So what's the point of setting it to 1 second here?

Reply:
After talking it through, we decided the timeout parameter for `_ChunkedTransaction()` is redundant. However, at this time, we did not find value in removing it. As it is not the subject of this PR.